API/API.mli

```diff
@@ -4669,8 +4669,6 @@ sig
   val constant_has_body : Declarations.constant_body -> bool
   val is_opaque : Declarations.constant_body -> bool
   val eq_recarg : Declarations.recarg -> Declarations.recarg -> bool
-  val body_of_constant :
```

Comment:
That one is incredibly unsafe, and has nothing to do in the kernel whatsoever. Furthermore, it is not used anymore in the plugins it seems, so by virtue of the miracle of the API semantics, it should not be there.